java/src/main/java/ai/rapids/cudf/DType.java

```diff
@@ -231,7 +231,7 @@ public String toString() {
    * @return DType
    */
   public static DType create(DTypeEnum dt) {
-    if (dt == DTypeEnum.DECIMAL32 || dt == DTypeEnum.DECIMAL64) {
+    if (DType.DECIMALS.contains(dt)) {
```

Comment:
I would really like to see an `isDecimalType` added to DTypeEnum (and have the DType method delegate accordingly) since `DECIMALS` is not public and code outside of cudf will want an easy/obvious way to check for decimal type on the enum directly without needing to construct a DType instance first.

Reply:
Done.